app/src/components/PeekPhase.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <Overlay>
+    <div class="PeekPile">
+      <Card
+        v-for="(card, i) in cards"
+        :key="card.id"
+        :type="card.type"
+        :text="card.text"
+        class="PeekPile__Card"
+        :style="{
+          '--card-offset-x': getOffset(i).x,
+          '--card-offset-y': getOffset(i).y
```

Comment:
What I like to do in this case is to just have an "index" variable and let the CSS do its own thing with it.
```html
<Card
  v-for="(card, i) in cards"
  :style="{  --peek-phase-card-index: i }"
/>
```

```scss
// adjust these variables
$interval-x: 10rem;
$interval-y: 2rem;
$offset-x: 5rem;
$offset-y: 0;

&__Card {
  transform: 
    translateX(calc(var(--peek-phase-card-index) * #{$interval-x} + #{$offset-x}))
    translateY(calc(var(--peek-phase-card-index) * #{$interval-y} + #{$offset-y}));
}
```